lang/en/docs/cli/environment.md

```diff
@@ -33,11 +33,15 @@ Software packages and programming libraries, are available through the **environ
 
 After loading the corresponding modules, the software can be used in [Job Scripts](../jobs-cli/batch-scripts/overview.md) for [Job execution via the CLI](../jobs-cli/overview.md). 
 
-## Python Environment
+## Default Python Environment
 
-By default, we implement **python 2.7.5** as our main system version. Different python versions can be installed via the the instructions and code contained in Ref. [^11].
+By default, we implement **Python 2.7.5** as our global system version. 
 
-Additionally, we explain how to create customized **python environments** [here](actions/create-python-env.md).
+If you'd like to use a more recent Python version, [we have **environment modules** available](modules.md) which you can activate to replace the global Python interpreter for an individual session.
```

Comment:
Same as above

Reply:
addressed in https://github.com/Exabyte-io/documentation/pull/181/commits/33bc51f6ccc4fff05f2e568f2ea2e32df0ccf3fa and https://github.com/Exabyte-io/documentation/pull/181/commits/a9f3c596c34e18f68e34c2aaacb24c279f2473b2